### PR TITLE
Issue1384 move files

### PR DESCRIPTION
**Pull request by lucasverleyen**
Fixes https://github.com/open-ideas/IDEAS/issues/1384.
Only minor changes.

### CODE REVIEW

File: IDEAS/Fluid/Taps/Examples/TapProfile.mo [hunk 1]
```diff
@@ -8,7 +8,7 @@ model TapProfile "Example with two 'TapProfile' models"
     annotation (Placement(transformation(extent={{-10,10},{10,30}})));
   IDEAS.Fluid.Taps.TapProfile tap2(redeclare package Medium = Medium, loadFile=
         Modelica.Utilities.Files.loadResource(
-        "modelica://IDEAS/Resources/DHW/DHW_1year_2adults_2children.txt"))
+        "modelica://IDEAS/Resources/Data/Fluid/Taps/DHW_1year_2adults_2children.txt"))
```
Comment (jelgerjansen) [inline IDEAS/Fluid/Taps/Examples/TapProfile.mo:11]: Please move the data to `modelica://IDEAS/Resources/Data/Fluid/Taps/Examples/` and update this string.

File: IDEAS/Fluid/Taps/TapProfile.mo [hunk 2]
```diff
@@ -6,7 +6,7 @@ model TapProfile
   parameter Real uniCon=1/60
     "Conversion factor to convert units from DHW profile [l/min] to [kg/s]";
   parameter String loadFile=Modelica.Utilities.Files.loadResource(
-    "modelica://IDEAS/Resources/DHW/DHW_1year_2adults.txt") annotation(Dialog(loadSelector(filter="All files (*.*)", caption="Select the DHW profile file")));
+    "modelica://IDEAS/Resources/Data/Fluid/Taps/DHW_1year_2adults.txt") annotation(Dialog(loadSelector(filter="All files (*.*)", caption="Select the DHW profile file")));
```
Comment (jelgerjansen) [inline IDEAS/Fluid/Taps/TapProfile.mo:9]: I would remove the default to make sure that users don't forget to include their own data file. An example of how to include their data can be found in `IDEAS.Fluid.Taps.Examples`.

**Review by jelgerjansen — CHANGES_REQUESTED** [review]

File: IDEAS/Fluid/Taps/Examples/TapProfile.mo (hunk 1 above)
Comment (lucasverleyen) [reply IDEAS/Fluid/Taps/Examples/TapProfile.mo:11]: good catch, I was too quick...

File: IDEAS/Fluid/Taps/TapProfile.mo (hunk 2 above)
Comment (lucasverleyen) [reply IDEAS/Fluid/Taps/TapProfile.mo]: https://github.com/open-ideas/IDEAS/pull/1393/commits/364f331f2697a877166e4659434b0c9b278f1986

**Review by jelgerjansen — APPROVED** [review]

**Pull request merged**